### PR TITLE
[WC-2018] fix: column hide by default not working

### PR DESCRIPTION
**Pull request by gjulivan**
### Pull request type
Bug Fix: this issue relates with previously introduced changed to allow non-numeric ID (fix a11y).
This changes causing "visible column" not working as expected due to the reference with "hidden column" that is still index-based.

### Description
Fix by adjusting correct attribute.

### CODE REVIEW

File: packages/pluggableWidgets/datagrid-web/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+
+-   We fixed an issue with column capabilities hide option that was unable to be hidden by default.
```
Comment (iobuhov) [inline packages/pluggableWidgets/datagrid-web/CHANGELOG.md:11]:
```suggestion
-   We fixed an issue with the column capabilities hide option that was unable to be hidden by default.
```

**Review by iobuhov — CHANGES_REQUESTED** [review]
LGTM, only changelog record needs update.

**Review by iobuhov — DISMISSED** [review]
LGTM

File: packages/pluggableWidgets/datagrid-web/cypress/integration/DataGridSelection.spec.js [hunk 2]
```diff
@@ -1,20 +1,3 @@
-function terminalLog(violations) {
```
Comment (leonardomendix) [inline packages/pluggableWidgets/datagrid-web/cypress/integration/DataGridSelection.spec.js:1]: *Extra: I have removed this unnecessary function leftover.

**Review by leonardomendix — APPROVED** [review]

**Review by iobuhov — APPROVED** [review]

**Pull request merged**